manifests/params.pp

```diff
@@ -6,7 +6,7 @@
   case $facts['os']['family'] {
     'RedHat': {
       $path = ['/usr/bin', '/bin']
-      $update_command = 'update-ca-trust enable && update-ca-trust'
+      $update_command = 'update-ca-trust'
```

Comment:
Why not
```suggestion
      $update_command = 'update-ca-trust extract'
```
as it's the documented "has to always work" invocation?

I'm fine either way, more curious about the decision.

Reply:
Simplicity was my motivation. I don't mind either way.

Reply:
From the Fedora man page (and EL9 is being updated to the same now):
```
COMMANDS
       (absent/empty command)
           Same as the extract command described below. (However, the command may print fewer warnings, as this command is being run during rpm package
           installation, where non-fatal status output is undesired.)

       extract
           Instruct update-ca-trust to scan the SOURCE CONFIGURATION and produce updated versions of the consolidated configuration files stored below the
           /etc/pki/ca-trust/extracted directory hierarchy.
```

Do we want warnings and errors? The resource is marked as `refreshonly` so the next Puppet run will simply skip over it. On the other hand, swallowing errors is not a good practice.

Reply:
I ended up changing to an explicit parameter.